android-core/src/main/java/com/mparticle/identity/MParticleIdentityClientImpl.java

```diff
@@ -75,18 +82,33 @@ public MParticleIdentityClientImpl(Context context, ConfigManager configManager,
 
     public IdentityHttpResponse login(IdentityApiRequest request) throws JSONException, IOException {
         JSONObject jsonObject = getStateJson(request);
+        IdentityHttpResponse existsResponse = checkIfExists(request, LOGIN_CALL);
+
+        if (existsResponse != null) {
+            return existsResponse;
+        }
+        Long maxAgeTime=86400L;
```

Comment:
Would this be better as a constant in the class instead of defined in each method?